src/site/content/en/blog/webtransport/index.md

```diff
@@ -20,104 +20,89 @@ feedback:
-
-### What's WebTransport?
-
-QuicTransport is one part of the larger [WebTransport proposal](https://wicg.github.io/web-transport/). WebTransport is a collection of APIs for sending and receiving data between a web client and a server. QuicTransport is the interface for using the QUIC protocol in the context of bi-directional WebTransport communications.
-
-Chrome is implementing the QuicTransport portion of WebTransport first, before any of the other proposed interfaces. The Chrome team made the decision to start with a QuicTransport after speaking to web developers about their use cases. We hope to solicit early feedback on the overall WebTransport effort based on developers' experiences with QuicTransport.
+The streams APIs, in contrast, provide [reliable](https://en.wikipedia.org/wiki/Reliability_(computer_networking)), ordered data transfer. They're [well-suited](https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#name-streams) to scenarios where you need to send or receive one or more streams of ordered data. Using multiple WebTransport streams is analogous to establishing multiple [TCP](https://en.wikipedia.org/wiki/Transmission_Control_Protocol) connections, but since HTTP/3 uses the lighter-weight [QUIC](https://www.chromium.org/quic) protocol under the hood, the can be opened and closed without as much overhead.
 
 ### Use cases
 
-This a small list of possible ways developers might use QuicTransport.
+This a small list of possible ways developers might use WebTransport.
 
 - Sending game state at a regular interval with minimal latency to a server via small, unreliable, out-of-order messages.
 - Receiving media streams pushed from a server with minimal latency, independent of other data streams.
 - Receiving notifications pushed from a server while a web page is open.
 
-As part of the origin trial process, we're interested in [hearing more](#feedback) about how you plan on using QuicTransport!
+As part of the origin trial process, we're interested in [hearing more](#feedback) about how you plan on using WebTransport!
+
+{% Aside %}
+Many of the concepts in this proposal were previously experimented with as part of the earlier [QuicTransport](https://chromestatus.com/feature/4854144902889472) Origin Trial, which did not end up being released as part of Chrome.
+
+WebTransport helps with similar use cases as QuicTransport, with the primary difference being that [HTTP/3](https://quicwg.org/base-drafts/draft-ietf-quic-http.html) is used as the underlying transport protocol, instead of using [QUIC](https://www.chromium.org/quic) directly.
```

Comment:
```suggestion
WebTransport helps with similar use cases as QuicTransport, with the primary difference being that [HTTP/3](https://quicwg.org/base-drafts/draft-ietf-quic-http.html) instead of [QUIC](https://www.chromium.org/quic) is the underlying transport protocol.
```